src/test/resources/wiremock/mappings/mapping-historicalPriceEmptyRange.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name" : "historical_price_empty_range",
+  "request" : {
+    "url" : "/stock/chart/max/{date}?symbol=AAPL&range=",
```

Comment:
What's this one for? I don't see a test that validates the behavior of passing an empty range.

Reply:
I had created a test for it but was having issues. I will look into doing it again and consult you if I have issues!